web_ui/src/pages/annotator/tools/ritm-tool/ritm-tool.component.tsx

```diff
@@ -142,7 +134,7 @@ export const RITMTool = ({ annotationToolContext }: ToolAnnotationContextProps):
         const newPoint = { x, y, positive };
         const points = result ? [...result.points, newPoint] : [newPoint];
 
-        execute(clampBox(currentBox, ROI), points, outputShape);
+        execute(image, clampBox(currentBox, ROI), points, outputShape);
```

Comment:
Wouldn't this load the same image every time we execute RITM? I feel like this is something we should avoid.

Reply:
It will, nice catch : ) . I reverted all refactors to keep them separate from this PR. Interface and loadImage restored. Please check the latest commit